pkg/config/tlscfg/testdata/gen-certs.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# The following commands were used to create the CA, server and client's certificates and keys in this directory used by unit tests.
+# These certificates use the Subject Alternative Name extension rather than the Common Name, which will be unsupported in Go 1.15.
+
+# Generate config files.
+# The server name (under alt_names in the ssl.conf) is `example.com`. (in accordance to [RFC 2006](https://tools.ietf.org/html/rfc2606))
+source gen-ssl-conf.sh example.com ssl.conf
+source gen-ssl-conf.sh wrong.com wrong-ssl.conf
```

Comment:
perhaps instead of writing to the current dir (and accidentally checking in), we could use temp dir
```shell
tmp_dir=$(mktemp -d -t certificates)
clean_up () {
    ARG=$?
    rm -rf $tmp_dir
    exit $ARG
} 
trap clean_up EXIT
```